views/course_menu_view.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <menuitem id="list_menu_courses"  name="Courses" sequence="2" action="open_academy_course_action"/>
```

Comment:
Plase add a root menu for the module.

```suggestion
    <menuitem id="open_academy_root_menu" name="Open Academy" sequence="1"/>
    <menuitem id="open_academy_course_menu"  name="Courses" sequence="1" action="open_academy_course_action" parent="open_academy_root_menu"/>
```
And see the [guideline](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action) to name views and menus